Tests/Fixtures/TestProject/Project.xcodeproj/xcshareddata/xcschemes/Framework.xcscheme

```diff
@@ -80,7 +80,7 @@
          <BuildableReference
             BuildableIdentifier = "primary"
             BlueprintIdentifier = "AE3F93DB94E7208F2F1D9A78"
-            BuildableName = "Framework.framework"
+            BuildableName = "Framework_iOS.framework"
```

Comment:
What made this change?

Reply:
Sorry, it's mistake. I reverted buildableName generation logic.